src/nextjs/server/cookies.ts

```diff
@@ -32,6 +47,9 @@ function getCookieStore(
 function getCookieStore(
   requestHeaders: ReturnType<typeof headers>,
   responseCookies: NextResponse["cookies"] | NextRequest["cookies"],
+  cookieConfig: {
+    maxAge: number | null;
+  },
```

Comment:
Are negative numbers valid values for `maxAge`? If not, I suggest throwing an error if one is provided.

Reply:
Ah, it looks like zero or negative means expire the cookie.

https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Set-Cookie#max-agenumber

Reply:
I'll still add validation, because I think setting all your cookies to immediately expire is not a valid configuration